samples/testing.fs

```diff
@@ -0,0 +1,45 @@
+module samples.unittest
```

Comment:
Nit: the file name and module name should be in `PascalCase` to match other samples